code/game/machinery/_machinery.dm

```diff
@@ -345,7 +345,7 @@
 		if(!(interaction_flags_machine & INTERACT_MACHINE_ALLOW_SILICON))
 			return FALSE
 
-	else if(isliving(user)) // If we are a living human
+	else if(ishuman(user)) // If we are a living human
```

Comment:
I believe it's better to check if the user is an advanced tool user instead of flipping the bird to every simple mob, dextrous or not.
```suggestion
	else if(ISADVANCEDTOOLUSER(user)) // If we are a living, dextrous mob
```

Reply:
you could check for the dextrous variable

Reply:
the checks for silicon are already done a few lines up, so your worry is unnecessary.

Reply:
Thanks for the suggestions! i tried to incorporate both. Let me know if you have any concerns